LICENSE

```diff
@@ -1,3 +1,14 @@
+These patches consist of multiple software modules and patches for Oracle
```

Comment:
`These patches consist of ... patches` I would simplify.
`This modules consists of patches for Oracle...`

Reply:
This is the exact wording from the external counsel after I had shown him my initial attempt.

Reply:
Ah, ok :-)